gatsby/content/projects/sdks/matrixbot-dotnet-sdk.mdx

```diff
@@ -0,0 +1,40 @@
+---
+layout: projectimage
+title: MatrixBot .NET Core SDK
+categories: 
+ - sdk
+description: A .NET 6.0 library to quickly interact with the matrix protocol
+author: enimatek-nl
+featured: true
```

Comment:
```suggestion
featured: false
```

We usually do not show newer projects as featured until they have some more widespread usage. Can you flip this to false so we merge it?

Reply:
ok, done